### PR TITLE
Remove batect and the many makefiles

### PR DESCRIPTION
**Pull request by lcardno10**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
 DATA_BUCKET=the-bucket
 RESOURCE_PREFIX=rapid
 DOMAIN_NAME=example.com
 COGNITO_USER_POOL_ID=11111111
 LAYERS=raw,layer
 CUSTOM_USER_NAME_REGEX="regex_expression"
-# SDK Specific
+
+## Build variables
+ECS_SERVICE=rapid-ecs-service
```
Comment (MotwaniM) [inline .env.example:16]: Just checking why we need these variables?

File: .github/.github.env [hunk 2]
```diff
@@ -5,3 +5,8 @@ LAYERS=raw,layer
 DOMAIN_NAME=example.com
 DATA_BUCKET=the-bucket
 CUSTOM_USER_NAME_REGEX=[a-zA-Z][a-zA-Z0-9@._-]{2,127}
+
+ECS_SERVICE=rapid-preprod-ecs-service
+ECS_CLUSTER=rapid-preprod-cluster
```
Comment (MotwaniM) [inline .github/.github.env:10]: As above

File: .github/workflows/release_api.yml [hunk 3]
```diff
@@ -46,30 +46,16 @@ jobs:
...
-        run: make ui-create-static-out
+        run: make ui/create-static-out
 
       - name: UI Zip and Release
         env:
           TAG: ${{ github.event.release.tag_name }}
           GH_TOKEN: ${{ github.token }}
-        run: make ui-zip-and-release tag=$TAG
-
-  cleanup:
```
Comment (MotwaniM) [inline .github/workflows/release_api.yml:63]: Just checking if we still need this? Presumably the docker images will start piling up on the runners longterm?

File: .github/workflows/release_sdk.yml [hunk 4]
```diff
@@ -37,21 +37,7 @@ jobs:
...
       - name: Setup Python Environment
         run: |
-          make sdk-setup
-          source sdk/.venv/bin/activate
+          make sdk/setup
 
       - name: SDK Release
-        run: make sdk-release
-
-  cleanup:
```
Comment (MotwaniM) [inline .github/workflows/release_sdk.yml:46]: As above but less relevant tbh

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -27,7 +27,7 @@ repos:
     hooks:
       - id: blacken-docs
   - repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
+    rev: 7.0.0
```
Comment (MotwaniM) [inline .pre-commit-config.yaml:30]: Not fully relevant here but worth updating checkov as well. Anticipate that will cause some checks to fail

File: Makefile [hunk 6]
```diff
@@ -1,190 +1,256 @@
...
+	TAG_NAME="$(UI_IMAGE_NAME)-$(GITHUB_SHORT_SHA)"
+else
+	TAG_NAME="$(UI_LATEST_TAG)-dev-$(GITHUB_SHORT_SHA)"
+endif
 
 .PHONY: help
 
+setup: brew precommit
+
+brew:				## Brew install all the dependencies
```
Comment (MotwaniM) [inline Makefile:32]: This is pretty neat!

**Review by MotwaniM — CHANGES_REQUESTED** [review]
Minor comments and checks

File: .env.example (hunk 1 above)
Comment (lcardno10) [reply .env.example:16]: We need them for a couple of the release commands, they do things like force redeploy the ECS service if you're releasing a new service and wait for the ECS service to be running

File: .github/workflows/release_api.yml (hunk 3 above)
Comment (lcardno10) [reply .github/workflows/release_api.yml:63]: Good shout

File: .github/workflows/release_sdk.yml (hunk 4 above)
Comment (lcardno10) [reply .github/workflows/release_sdk.yml:46]: Yeah, here there is no container creation so we're fine without the cleanup

**Pull request merged**